storybook/stories/index.js

```diff
@@ -115,46 +117,20 @@ storiesOf('Components/Fee Slider', module)
     />
   ))
 
-storiesOf('Components/Fetching Indicators', module)
-  .addParameters({
-    options: {
-      hierarchySeparator: /\//,
-      hierarchyRootSeparator: /\|/,
-    },
-  })
-  .addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-  .add('FetchingIndicator FETCHING', () => (
-    <FetchingIndicator status='FETCHING' />
-  ))
-  .add('FetchingIndicator SYNCING', () => (
-    <FetchingIndicator status='SYNCING' />
-  )).add('FetchingIndicator SYNCED', () => (
-    <FetchingIndicator status='SYNCED' />
-  ))
-
 storiesOf('Components/Labels', module)
   .addParameters({
     options: {
       hierarchySeparator: /\//,
       hierarchyRootSeparator: /\|/,
     },
   })
-  .addDecorator(getStory => <CenterView>{getStory()}</CenterView>)
-  .add('Label 1', () => (
-    <Label labelTextAlign='left' labelType='currencyColored' text='FIRST LABEL' />
-  ))
-  .add('Label 2', () => (
-    <Label labelTextAlign='right' text='SECOND LABEL' />
+  .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
+  .add('Labels', () => (
+    <React.Fragment>
+      <Label labelTextAlign='left' labelType='currencyColored' text='FIRST LABEL' />
```

Comment:
Please replace `FIRST LABEL` and 'SECOND LABEL` to informative text:
1) Currency colored  left aligned
2) Pure right aligned label